app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonViewModel.kt

```diff
@@ -0,0 +1,89 @@
+package org.oppia.android.app.topic.lessons
+
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
+import androidx.lifecycle.ViewModel
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.app.model.ProfileId
+import org.oppia.android.app.model.StorySummary
+import org.oppia.android.app.model.Topic
+import org.oppia.android.domain.oppialogger.OppiaLogger
+import org.oppia.android.domain.topic.TopicController
+import org.oppia.android.util.data.AsyncResult
+import org.oppia.android.util.data.DataProviders.Companion.toLiveData
+import javax.inject.Inject
+
+/** [ViewModel] for [TopicLessonsFragment]. */
+@FragmentScope
+class TopicLessonViewModel @Inject constructor(
+  private val fragment: Fragment,
+  private val oppiaLogger: OppiaLogger,
+  private val topicController: TopicController
+) {
+  private var internalProfileId: Int = -1
+  private lateinit var topicId: String
+  private lateinit var storyId: String
+  lateinit var topicStoryList: List<StorySummary>
```

Comment:
This can be `private`

Reply:
accessing this is presenter file, can't be private.